### ISSUE TITLE
Should HP:6000108 be generalized to include females?

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10940]
**HPO term**
https://hpo.jax.org/browse/term/HP:6000108

**Suggested revision of textual definition**
The definition has "Fluid that originates from the urethra"  but the term name is male-specific "Abnormal penile discharge".  

Should this finding be generalized to include females, or a female version created, or a parent urethral discharge term to include both males and females?

**Comment by pnrobinson** [comment]
Agree. I have changed the lanbel to abnormal urethral discharge and moved the term accordingly.
